packages/typespec-vscode/src/vscode-cmd/emit-code/emit-code.ts

```diff
@@ -134,17 +140,14 @@ async function doEmit(mainTspFile: string, emitter: Emitter) {
     return;
   }
 
+  if (emitters.length === 0) {
+    logger.info("No emitter. Generating skipped.");
```

Comment:
not really this pr problem but the wording should be changed. `generating` is not a concept exposed in typespec. We mention `emitting`, `emitter(s)` we should probably stick to the correct terminology.

Reply:
Thanks @timotheeguerin  We need to ensure consistency in terminology for both the CLI and VSCode extension. I will update all the terminology together after we sync up and finalize the terms, possibly in another PR.